### ISSUE TITLE
world not loading

### ISSUE THREAD

**Issue by Casper1123** [SleepyTrousers/EnderIO#4858]
#### Issue Description:
so i am playing this singleplayer mod thingy, and my game crashed and i tried to reload it. this didnt work for 30 minutes 5x. so i reported this on the minecraft forge forums, as there i thought i could get the awnser ;
  http://www.minecraftforge.net/forum/topic/66266-world-not-loading/?tab=comments#comment-317393

they said it was to do with EnderIO and RefinedStorage (2 of my favorite mods, so i cant cut them out).

#### What happens:
world stops loading, all the crash log info and log's are on the miencraft forge forum

#### What you expected to happen:
just for the world to load, or for it give an error in the output log.

#### Steps to reproduce:

1. load world
2. wait
3. wait
… oh shit
i think this is a personal problem, but i would like a fix as most of my machines run on EnderIO components

____
#### Affected Versions (Do *not* use "latest"):

- EnderIO: 1.10.203.1.193
- EnderCore: 1.10.2-0.4.1.66-beta
- Minecraft: 1.10.2
- Forge: 1.10.2-forge1.10.2-12.18.3.2442
- SpongeForge? yes/no No
- Optifine? yes/no Ye
- Single Player and/or Server? Singleplayer

#### Your most recent log file where the issue was present: 
see minecraftforge forum post in the link above

**Comment by HenryLoenwind** [comment]
![image](https://user-images.githubusercontent.com/1485873/45042098-35c62480-b06a-11e8-8e65-f1760770748e.png)

**Comment by Casper1123** [comment]
When im back home ill post the log files here.

**Comment by Casper1123** [comment]
Didnt know it got removed

**Comment by Casper1123** [comment]
[latest.log](https://github.com/SleepyTrousers/EnderIO/files/2352956/latest.log)

the latest log

**Comment by HenryLoenwind** [comment]
```
java.lang.NullPointerException: Tesselating block model
	at com.raoulvdberge.refinedstorage.block.BlockBase.getActualState(BlockBase.java:76)
	at com.raoulvdberge.refinedstorage.block.BlockNode.getActualState(BlockNode.java:49)
	at net.minecraft.block.state.BlockStateContainer$StateImplementation.getActualState(BlockStateContainer.java:414)
	at crazypants.enderio.material.fusedQuartz.BlockFusedQuartz.shouldSideBeRendered(BlockFusedQuartz.java:100)
	at net.minecraft.block.state.BlockStateContainer$StateImplementation.shouldSideBeRendered(BlockStateContainer.java:424)
	at team.chisel.ctm.client.util.CTMLogic.isConnected(CTMLogic.java:365)
	at team.chisel.ctm.client.util.Dir.isConnected(Dir.java:84)
	at team.chisel.ctm.client.util.CTMLogic.buildConnectionMap(CTMLogic.java:215)
	at team.chisel.ctm.client.util.CTMLogic.createSubmapIndices(CTMLogic.java:161)
	at team.chisel.ctm.client.texture.ctx.TextureContextCTM.<init>(TextureContextCTM.java:30)
	at team.chisel.ctm.client.texture.type.TextureTypeCTM.getBlockRenderContext(TextureTypeCTM.java:24)
	at team.chisel.ctm.client.texture.type.TextureTypeCTM.getBlockRenderContext(TextureTypeCTM.java:14)
	at team.chisel.ctm.api.util.RenderContextList.<init>(RenderContextList.java:34)
	at team.chisel.ctm.client.state.ChiselExtendedState.getContextList(ChiselExtendedState.java:76)
	at team.chisel.ctm.client.model.AbstractCTMBakedModel.func_188616_a(AbstractCTMBakedModel.java:208)
	at net.minecraft.client.renderer.BlockModelRenderer.func_187498_b(BlockModelRenderer.java:102)
	at net.minecraftforge.client.model.pipeline.ForgeBlockModelRenderer.func_187498_b(ForgeBlockModelRenderer.java:107)
	at net.minecraft.client.renderer.BlockModelRenderer.func_187493_a(BlockModelRenderer.java:72)
	at net.minecraft.client.renderer.BlockModelRenderer.func_178267_a(BlockModelRenderer.java:52)
	at net.minecraft.client.renderer.BlockRendererDispatcher.func_175018_a(BlockRendererDispatcher.java:81)
	at codechicken.lib.render.block.CCExtendedBlockRendererDispatcher.func_175018_a(CCExtendedBlockRendererDispatcher.java:54)
	at gcewing.architecture.BaseAORenderingManager$CustomBlockRendererDispatcher.func_175018_a(BaseAORenderingManager.java:85)
	at codechicken.lib.render.block.CCBlockRendererDispatcher.func_175018_a(CCBlockRendererDispatcher.java:61)
	at net.minecraft.client.renderer.chunk.RenderChunk.func_178581_b(RenderChunk.java:292)
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.func_178474_a(SourceFile:100)
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.run(SourceFile:43)
	at java.lang.Thread.run(Thread.java:745)
```

=> refinedstorage

**Comment by Casper1123** [comment]
So if it depends on refined, it might be smart to get inti contact with the devs, do you have any idea on how to?

**Comment by HenryLoenwind** [comment]
https://github.com/raoulvdberge/refinedstorage/

**Comment by raoulvdberge** [comment]
Closed this on RS' side as 1.10 isn't maintained anymore.

**Comment by Casper1123** [comment]
but why. i cant play with your awsome mod anymore just because of that bug. my creative world also broke in the same way. i really cant understand why you leave players with a little bit of an older version behind, when players that play on 1.11 or 1.12 can get the same issue and then they do get help. and your response was not being INTERESTED, wich is completely something else.
but most likely you are going to ignore me, so thanks for ruining my birthday project
